galgebra/printer.py

```diff
@@ -801,10 +801,24 @@ def repl_func(m):
 
 
 def tex(paper=(14, 11), debug=False, prog=False, pt='10pt'):
-    """
+    r"""
     Post processes LaTeX output (see comments below), adds preamble and
     postscript.
 
+    This postprocessing includes:
+
+    * ``##`` at the start of a line toggles postprocessing on and off. It
+      begins as on. The rest of the line is never processed.
```

Comment:
"toggle on and off" and "it begins as on" together is confusing.

Reply:
Better as https://github.com/pygae/galgebra/pull/379#discussion_r429364677?